lib/src/test/kotlin/com/lemonappdev/konsist/core/verify/assert/snippet/assert-not-check-declaration-with-suppress-annotation-with-konsist-prefix.kttxt

```diff
@@ -0,0 +1,5 @@
+class SampleClassText {
```

Comment:
1. All snippet names should start with `assert`, not `assert-not`
2. Names should 

1.
snippet:
```kotlin
class SampleClassText {
    @Suppress("assert-not-check-declaration-with-suppress-annotation-with-konsist-prefix")
    fun sampleFunctionWithIncorrectSuffix() {
    }
}

```
test names: `assert-supress-by-name-at-declaration-level` / `assert-supress-by-konsist-and-name-at-declaration-level`

2. 
snippet:
```kotlin
@Suppress("konsist.assert-not-check-nested-declarations-when-top-level-declaration-has-suppress-annotation-with-konsist-prefix")
class SampleClassWithIncorrectSuffix {
    fun sampleFunctionWithIncorrectSuffix() {}
}

```

test names: `assert-supress-by-name-at-declaration-parent-level` / `assert-supress-by-konsist-and-name-at-declaration-parent-level`

3. 
snippet
```kotlin
@file:Suppress("assert-not-check-declarations-in-file-with-suppress-annotation-without-konsist-prefix")

class SampleClassWithIncorrectSuffix {
    fun sampleFunctionWithIncorrectSuffix(){}
}

```

test names: `assert-supress-by-name-at-file-level` / `assert-supress-by-konsist-and-name-at-file-level`